site/en/codelabs/silabs-openthread-hardware/index.lab.md

```diff
@@ -0,0 +1,438 @@
+
+1. Demos: (Easier and recommended) Flash the prebuilt demos directly on the radio board for any of the sample applications.
+
+   OR
+
+2. Projects: Create, build and flash a sample application. This option allows you to customize the application in the project.
+
+#### Demos
+
+Duration: 5 minutes
+
+Demos are prebuilt firmware images that are ready to download to a compatible device. The quickest way to find if a demo is available for your part in Simplicity Studio is by clicking on your part under Debug Adapters view and then navigating to the EXAMPLE PROJECTS & DEMOS tab in the Launcher Perspective. Disable the Example Projects filter and check the Thread radio checkbox under Technology Type.
+
+![Studio demos](./img/Silabs/studio_demos.png)
+
+Precompiled demo application images provided with the OpenThread SDK are compatible with the following boards: 
+
+1. BRD4161a
+
+2. BRD4166a
+
+3. BRD4168a
+
+4. BRD4180a
+
+5. BRD4304a
+
+   This list may be updated in the future SDK releases to include more radio boards. Please refer to the Silicon Labs OpenThread SDK release notes under Documentation for a full list of supported parts.
+
+##### Flash the demos
+
+Flash the following demos on the respective boards. To flash, select your board under Debug Adapters on the left and click on RUN for the corresponding sample applications. A pop up window will show the flash progress.
+
+1. BRD4166A: **ot-rcp**
+
+   This device will function as a Radio Co Processor to the OT Border Router. We will use this device to create a Thread network and onboard the other two devices on the Thread network. Being a Border Router this device also acts as a gateway for the devices in the Thread network to communicate over the internet.
+
+2. Two BRD4168A: **ot-cli-ftd**
+
+   These two devices will act as Full Thread Devices. They will join the Thread network created by the OTBR.
+
+If you use the Demos option to flash the radio boards then you can skip the next section about creating a project.
+
+#### Creating a Project Based on an Example
+
+Duration: 10 minutes
+
+We will create two projects. The ot-rcp project for BRD4166A and the ot-cli-ftd project for the two BRD4168A. Please follow these steps and select the appropriate sample application for your board.
+
+1. Open Studio's File menu and select New > Silicon Labs Project Wizard. The Target, SDK, and Toolchain Selection dialog opens. Do not change the default Simplicity IDE / GNU toolchain supported by OpenThread. Click Next.
+
+   Target boards: Shows the selected radio board (BRD4168A) along with the main board (BRD4001A)
+
+   Target Device: This field shows the microcontroller chip (MCU) onboard. The BRD4168A has an EFR32MG13 MCU onboard.
+
+   SDK: Here you can select the SDK version of OT that you are working with. For this codelab we select OpenThread 1.1.2.0
+
+   IDE/ Toolchain: The toolchain that will be used for compiling the OT project. We use GNU ARM 7.2.1
```

Comment:
These should be formatted as an indented list under the 1. bullet:

* Target boards: Shows the selected radio board (BRD4168A) along with the main board (BRD4001A)
* Target Device: This field shows the microcontroller chip (MCU) onboard. The BRD4168A has an EFR32MG13 MCU onboard.
* SDK: Here you can select the SDK version of OT that you are working with. For this codelab we select OpenThread 1.1.2.0
* IDE/ Toolchain: The toolchain that will be used for compiling the OT project. We use GNU ARM 7.